pkg/traceql/engine_metrics_test.go

```diff
@@ -2643,7 +2643,7 @@ func BenchmarkSumOverTime(b *testing.B) {
 	for range totalSpans {
 		s := time.Duration(randInt(1, 3)) * time.Second
 		v := randFloat(minimum, maximun)
-		in = append(in2, newMockSpan(nil).WithStartTime(uint64(s)).WithSpanString("foo", "bar").WithSpanFloat("kafka.lag", v).WithDuration(100))
+		in = append(in, newMockSpan(nil).WithStartTime(uint64(s)).WithSpanString("foo", "bar").WithSpanFloat("kafka.lag", v).WithDuration(100))
```

Comment:
i don't know what in2 was but is this a behavior change?

Reply:
This is constructing test data for a benchmark, and it looks like it was doing it incorrectly before.  Oh you mean in regards to linting:  yes.  But it was caught by the linter